### PR TITLE
Add Japanese translation to text

### PR DESCRIPTION
**Pull request by yuriamm**
Added Japanese translation for what was implemented in https://github.com/elixirschool/school_house/pull/192. 
Some are referred to the existing translations in "lessons".
Also updated some small parts in the localization as well. (i.e. added missing gettext)

Contributors: @yuriamm @kuunamaka @OR-2000

### CODE REVIEW

**Review by doomspork — APPROVED** [review]

File: priv/gettext/de/LC_MESSAGES/default.po [hunk 1]
```diff
@@ -411,6 +411,11 @@ msgstr ""
 msgid "Date"
 msgstr ""
 
+#, elixir-format
+#: lib/school_house_web/templates/layout/_lesson_menu.html.heex:312
+msgid "Debugging"
+msgstr ""
```
Comment (kinson) [inline priv/gettext/de/LC_MESSAGES/default.po:417]: I think this file needs an update here as well to `"on Twitter."`

https://github.com/elixirschool/school_house/blob/3bdcac9f5128329f80e6aac0475291e6e6c13a1c/priv/gettext/de/LC_MESSAGES/default.po#L936-L939

Comment (yuriamm) [reply priv/gettext/de/LC_MESSAGES/default.po:417]: Good catch! Fixed in https://github.com/elixirschool/school_house/pull/197/commits/9fb2dfd058cff81fb5738174192eb5aff22d61a5

**Comment by doomspork** [comment]
Thank you and welcome @yuriamm! I'll let @kinson take another peek to see if he catches anything and then we can merge!

**Review by kinson — APPROVED** [review]
🚀

**Pull request merged**